hosts/ficolobuild/configuration.nix

```diff
@@ -36,4 +36,10 @@
   # Use the systemd-boot EFI boot loader.
   boot.loader.systemd-boot.enable = true;
   boot.loader.efi.canTouchEfiVariables = true;
+
+  # Connect to public servers.
```

Comment:
Contrary to the comment, this is about connecting to "private" IP instead of "public" one (which you would get from the dns)